img/t/ticketsource.co.uk.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="123.743" height="130.414"><g fill="#eb601a" fill-rule="evenodd"><path d="m121.919 61.915-24.37 6.44c4.134 15.642-5.23 31.73-20.872 35.865-15.642 4.134-31.731-5.23-35.865-20.872 0 0-24.276 6.796-24.37 6.44 7.697 29.126 37.549 46.498 66.676 38.802 29.125-7.697 46.498-37.55 38.801-66.675"/><path d="m6.44 52.58 41.19-10.885 12.438 47.06 24.37-6.44-12.437-47.06 41.192-10.885L106.753 0 0 28.21z"/></g></svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 123.7 130.4"><g fill="#eb601a" fill-rule="evenodd"><path d="m121.919 61.915-24.37 6.44c4.134 15.642-5.23 31.73-20.872 35.865-15.642 4.134-31.731-5.23-35.865-20.872 0 0-24.276 6.796-24.37 6.44 7.697 29.126 37.549 46.498 66.676 38.802 29.125-7.697 46.498-37.55 38.801-66.675"/><path d="m6.44 52.58 41.19-10.885 12.438 47.06 24.37-6.44-12.437-47.06 41.192-10.885L106.753 0 0 28.21z"/></g></svg>
```

Here's an SVG that uses `viewBox` instead of `width` and `height`